CHANGELOG.md

```diff
@@ -2,11 +2,11 @@
 
 ## 4.0.0
 
-- `DataResourceType`, `JSONDictionaryResourceType` and `JSONArrayResourceType` now throw on failure
-- Renamed function on `DataResourceType` from `func result(from data: Data) -> Result<Model>` to `func model(from data: Data) throws -> Model`
```

Comment:
I removed because it only referred to the `DataResourceType` but since you mentioned it I'll add it back and include the other functions changed for the `JSONDictionaryResourceType` and `JSONArrayResourceType`